packages/plugin-cloudflare/scripts/postinstall.js

```diff
@@ -140,6 +147,17 @@ function sha256(filePath) {
   return createHash('sha256').update(fileBuffer).digest('hex')
 }
 
+function isCurrentVersionOlderThanExpected(current, expected) {
+  const currentParts = current.split('.')
+  const expectedParts = expected.split('.')
+  for (let i = 0; i < currentParts.length; i++) {
+    if (parseInt(expectedParts[i]) > parseInt(currentParts[i])) {
+      return true
+    }
+  }
+  return false
+}
+
```

Comment:
could we use  `semver.gt` instead? we already have it in the monorepo

Reply:
done, wasn't sure about adding a new dependency here but it's 👌